tests/testthat/test-as_tibble.R

```diff
@@ -0,0 +1,193 @@
+t1 <- trial |>
+  tbl_summary()
+
+t2 <- glm(response ~ age + grade, trial, family = binomial(link = "logit")) |>
+  tbl_regression(exponentiate = TRUE)
+
+t3 <- tbl_uvregression(
+  trial |> dplyr::select(response, age, grade),
+  method = glm,
+  y = response,
+  method.args = list(family = binomial),
+  exponentiate = TRUE
+)
```

Comment:
for these objects we use again and again, can we please name them in a way that helps us identify what kind of table they are, e.g. `t1_summary`, `t2_regression`?